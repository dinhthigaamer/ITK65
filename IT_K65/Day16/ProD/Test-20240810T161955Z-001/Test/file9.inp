498
-395982085 -395017479 561184628 -808745694 -610250791 -662539592 -381980660 -117822510 -336748306 915281835 -592835995 994345349 -387971408 471069644 447340673 220946140 -558652982 -369472299 -968026972 296115471 553911994 -277260882 423956902 559528011 306614424 -315883078 -716709374 -870539707 -173275941 632753517 -174465715 -491927737 -537866690 -516653927 -505429066 -696966991 531378532 591290442 126602354 -254379797 997123275 -97396031 653006525 466129541 -627203107 601672708 620260347 -601710375 -767535937 349051125 -280821831 844185267 -975801523 286351005 -282981850 -865632795 -699454841 -975203418 488447770 310085725 746852621 -289554219 -982369764 600285768 561577340 258991071 -569805979 -372655165 -106685554 -619607960 188856473 -268960803 886429660 -587284372 911396803 -462990289 -228681210 -876239914 -822830527 -750131315 -401234943 116991909 -582713480 979631847 813552478 627922193 -45687378 360503305 -822034236 262588536 -419476433 -154657464 420790839 865129404 12226550 864017753 734626475 -676940469 -967951504 -550438390 792998079 497736429 89558819 702307983 -451933763 -650808132 -427479204 -287677771 147901369 611343200 -566550401 -37621397 -111903730 537204219 -102377750 280860753 720177616 718644971 -441669012 297802119 454468308 -372346475 -312743853 129511318 171938003 -916343748 -538159488 -312982768 764798663 -517114967 652111603 -967965005 -305746781 -611649380 -159816738 900825283 838988243 -541258237 -485855065 114562516 -948464303 -530958263 -245146892 182333234 -717676994 535373036 553582965 813272520 -54683190 458948622 -572944777 -641808336 540052700 375814040 643870148 631090345 -290298628 -247146138 -63005588 -738608773 811192913 319109217 -309363053 -308214793 -277175318 -832000515 259159813 -461222146 -268447945 -53315011 -940672579 -509259502 -655809292 -886270423 -832995027 69112690 279850608 726838739 -431610610 242749870 -244020223 -396865433 129534799 182767854 -939698671 342579400 330027106 893561388 -263513962 643369418 276245179 -99206565 -473767261 -168908664 -853755837 -270920164 -661186118 199341055 765543053 908211534 -276083750 964570817 666517330 861262418 793274472 -869892143 -947504130 120933217 -791095281 -275130071 -862472296 -419121409 719891051 718174936 -140362196 -669193674 -598621021 -464130929 450701419 -331183926 107203430 946959233 -565825915 -558199897 246556991 -666216081 295568243 913485902 -698885084 -231129961 -821100134 975996141 -569670625 -648055716 838640930 -667481805 151843048 -163522445 854497893 944119065 -416237303 -104981166 -443170755 -281858820 -950834842 291469402 967280473 874946031 -957297780 -452549562 -985832391 -785408930 565954935 -853063307 -319109874 830127242 499844561 -15208312 -912434954 694370765 30287211 -301498724 -574631061 782757575 -896623404 -452924198 427523683 -174566784 -456599891 -135026394 742964321 824791097 -639651556 506059551 -961596764 -172600836 169054244 765909567 476174133 -830432248 819713950 546086496 -120572479 -453754913 866708094 -352901591 -629650253 -837883499 743403332 -675238615 904381060 -619204583 -863012565 573615531 89655747 -303895833 255160827 458421076 876776163 -9891392 81864965 59586643 -500183411 201469279 -388483857 -771791546 375605886 -401436354 163676168 -919764203 216180796 73194154 -678524520 828156771 -658794885 -180050736 573049200 946780480 344117709 758834059 422097166 624419877 -214569246 621043936 532111328 204116679 -222500229 744636077 675369992 -804670105 500277302 459384928 217539875 -570819339 893429283 980626314 -547750343 -508275905 52238321 -254015726 317272932 -73792744 754681324 11987232 448560012 -432133135 701761976 -773572096 -652462807 -611864714 -194856103 -508667693 -609016585 -999697069 -182966761 -111324462 -472839603 720930032 296436382 -744701696 412348152 -394293536 287505711 700413459 -14155038 -546121909 -786280211 883000528 585252503 20555751 -311185792 -204227491 177362566 428445161 -834363137 174355886 -575335658 341930626 6418360 -650239450 -661861673 -377097463 -881455594 -456667909 313428016 -696593785 38987748 -152892436 793857487 -706805491 297085276 404541133 794415284 210975666 -582977877 162762809 -442456977 -127196077 -661379198 -294430078 264979559 307743522 999377708 -716372338 404010039 -112369236 -896455676 -551836219 -465476848 803513872 939678924 636461672 -811241152 -382628521 323937921 148251899 560281551 -485017554 573120686 -201614339 -496654218 -745896123 -901023033 -118388110 -324987651 -702053438 468613602 -508569356 103347774 478352442 -816585229 -11228888 -490572558 -658018749 495456513 349245791 -257730664 39091727 -52864920 -921018084 -324720122 12451753 765011251 -244015179 -306842242 -851505141 -101055072 290371548 -772728152 -392296812 596334263 -388237397 76592284 -31374821 -682754412 -96639618 -787366638 364346554 -204260884 138595284 716932205 -982672609 -275030064 -938677670 -39740431 -443848985 -740241884 618874 -232452273 -552546788 -566077881 -39621850 -779309401 60567503 -831057568 -59891439 126385546 -594977402 568193475 809690331 -278787049 675831640 -286054588 -970118831 289709116 792515998 -517352546 784967129 -290842433 -341967853 585293768 892578873 -324993397 -642007741 -163248122 310422151 995947772 -381191262 
-395982085 -395017479 561184628 -808745694 -610250791 -662539592 -381980660 -117822510 -336748306 915281835 -592835995 994345349 -387971408 471069644 447340673 220946140 -558652982 -369472299 -968026972 296115471 553911994 -277260882 423956902 559528011 306614424 -315883078 -716709374 -870539707 -173275941 632753517 -174465715 -491927737 -537866690 -516653927 -505429066 -696966991 531378532 591290442 126602354 -254379797 997123275 -97396031 653006525 466129541 -627203107 601672708 620260347 -601710375 -767535937 349051125 -280821831 844185267 -975801523 286351005 -282981850 -865632795 -699454841 -975203418 488447770 310085725 746852621 -289554219 -982369764 600285768 561577340 258991071 -569805979 -372655165 -106685554 -619607960 188856473 -268960803 886429660 -587284372 911396803 -462990289 -228681210 -876239914 -822830527 -750131315 -401234943 116991909 -582713480 979631847 813552478 627922193 -45687378 360503305 -822034236 262588536 -419476433 -154657464 420790839 865129404 12226550 864017753 734626475 -676940469 -967951504 -550438390 792998079 497736429 89558819 702307983 -451933763 -650808132 -427479204 -287677771 147901369 611343200 -566550401 -37621397 -111903730 537204219 -102377750 280860753 720177616 718644971 -441669012 297802119 454468308 -372346475 -312743853 129511318 171938003 -916343748 -538159488 -312982768 764798663 -517114967 652111603 -967965005 -305746781 -611649380 -159816738 900825283 838988243 -541258237 -485855065 114562516 -948464303 -530958263 -245146892 182333234 -717676994 535373036 553582965 813272520 -54683190 458948622 -572944777 -641808336 540052700 375814040 643870148 631090345 -290298628 -247146138 -63005588 -738608773 811192913 319109217 -309363053 -308214793 -277175318 -832000515 259159813 -461222146 -268447945 -53315011 -940672579 -509259502 -655809292 -886270423 -832995027 69112690 279850608 726838739 -431610610 242749870 -244020223 -396865433 129534799 182767854 -939698671 342579400 330027106 893561388 -263513962 643369418 276245179 -99206565 -473767261 -168908664 -853755837 -270920164 -661186118 199341055 765543053 908211534 -276083750 964570817 666517330 861262418 793274472 -869892143 -947504130 120933217 -791095281 -275130071 -862472296 -419121409 719891051 718174936 -140362196 -669193674 -598621021 -464130929 450701419 -331183926 107203430 946959233 -565825915 -558199897 246556991 -666216081 295568243 913485902 -698885084 -231129961 -821100134 975996141 -569670625 -648055716 838640930 -667481805 151843048 -163522445 854497893 944119065 -416237303 -104981166 -443170755 -281858820 -950834842 291469402 967280473 874946031 -957297780 -452549562 -985832391 -785408930 565954935 -853063307 -319109874 830127242 499844561 -15208312 -912434954 694370765 30287211 -301498724 -574631061 782757575 -896623404 -452924198 427523683 -174566784 -456599891 -135026394 742964321 824791097 -639651556 506059551 -961596764 -172600836 169054244 765909567 476174133 -830432248 819713950 546086496 -120572479 -453754913 866708094 -352901591 -629650253 -837883499 743403332 -675238615 904381060 -619204583 -863012565 573615531 89655747 -303895833 255160827 458421076 876776163 -9891392 81864965 59586643 -500183411 201469279 -388483857 -771791546 375605886 -401436354 163676168 -919764203 216180796 73194154 -678524520 828156771 -658794885 -180050736 573049200 946780480 344117709 758834059 422097166 624419877 -214569246 621043936 532111328 204116679 -222500229 744636077 675369992 -804670105 500277302 459384928 217539875 -570819339 893429283 980626314 -547750343 -508275905 52238321 -254015726 317272932 -73792744 754681324 11987232 448560012 -432133135 701761976 -773572096 -652462807 -611864714 -194856103 -508667693 -609016585 -999697069 -182966761 -111324462 -472839603 720930032 296436382 -744701696 412348152 -394293536 287505711 700413459 -14155038 -546121909 -786280211 883000528 585252503 20555751 -311185792 -204227491 177362566 428445161 -834363137 174355886 -575335658 341930626 6418360 -650239450 -661861673 -377097463 -881455594 -456667909 313428016 -696593785 38987748 -152892436 793857487 -706805491 297085276 404541133 794415284 210975666 -582977877 162762809 -442456977 -127196077 -661379198 -294430078 264979559 307743522 999377708 -716372338 404010039 -112369236 -896455676 -551836219 -465476848 803513872 939678924 636461672 -811241152 -382628521 323937921 148251899 560281551 -485017554 573120686 -201614339 -496654218 -745896123 -901023033 -118388110 -324987651 -702053438 468613602 -508569356 103347774 478352442 -816585229 -11228888 -490572558 -658018749 495456513 349245791 -257730664 39091727 -52864920 -921018084 -324720122 12451753 765011251 -244015179 -306842242 -851505141 -101055072 290371548 -772728152 -392296812 596334263 -388237397 76592284 -31374821 -682754412 -96639618 -787366638 364346554 -204260884 138595284 716932205 -982672609 -275030064 -938677670 -39740431 -443848985 -740241884 618874 -232452273 -552546788 -566077881 -39621850 -779309401 60567503 -831057568 -59891439 126385546 -594977402 568193475 809690331 131554320 675831640 -286054588 -970118831 289709116 792515998 -517352546 784967129 -290842433 -341967853 585293768 892578873 -324993397 -642007741 -163248122 310422151 995947772 -381191262 