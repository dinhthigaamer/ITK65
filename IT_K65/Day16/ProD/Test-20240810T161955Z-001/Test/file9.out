-12989002921828
